src/main/java/com/intelliChef/view/ViewManager.java

```diff
@@ -4,21 +4,39 @@
 import com.intelliChef.entities.Ingredient;
 import com.intelliChef.use_case.IngredientRepository;
 import com.intelliChef.use_case.analyzeImage.AnalyzeImageInteractor;
-import com.intelliChef.entities.Recipe;
+import com.intelliChef.adapters.presentation.RecipeListView;
+import com.intelliChef.adapters.presentation.RecipeDetailView;
+import com.intelliChef.use_case.ports.input.LoadRecipesUseCase;
+import com.intelliChef.use_case.ports.input.SelectRecipeUseCase;
+import com.intelliChef.use_case.ports.input.NavigationUseCase;
+import com.intelliChef.frameworks.factories.RecipeViewFactory;
 
 import java.util.List;
 
 public class ViewManager implements NavigationCall {
     private final ViewFactory viewFactory;
     private final AnalyzeImageInteractor analyzeImageInteractor;
     private IngredientRepository ingredientRepository;
-    private final RecipeViewFactory recipeviewFactory;
+    private final RecipeViewFactory recipeViewFactory;
+    private final LoadRecipesUseCase loadRecipesUseCase;
+    private final SelectRecipeUseCase selectRecipeUseCase;
+    private final NavigationUseCase navigationUseCase;
+    private RecipeListView currentRecipeListView;
+    private RecipeDetailView currentRecipeDetailView;
 
-
-    public ViewManager(ViewFactory viewFactory, AnalyzeImageInteractor analyzeImageInteractor, RecipeViewFactory recipeviewFactory) {
+    public ViewManager(
+            ViewFactory viewFactory,
+            AnalyzeImageInteractor analyzeImageInteractor,
+            RecipeViewFactory recipeViewFactory,
+            LoadRecipesUseCase loadRecipesUseCase,
```

Comment:
Where are these initialized? Is it necessary for them to be passed as parameters here?